issue_template.md

```diff
@@ -0,0 +1,36 @@
+<!--- Provide a general summary in the title above this comment-->
```

Comment:
Add "Help avoid duplicate issue reports" part

I searched for existing issues that already report this problem:
https://monosnap.com/file/jrSemsSxCGMZedRJE7jHwqbayWOqP8
"